sdk/openai/Azure.AI.OpenAI/tests/OpenAIInferenceTests.cs

```diff
@@ -18,145 +17,94 @@ public OpenAIInferenceTests(bool isAsync)
         {
         }
 
-        /// <summary>
-        /// Test an instance of OpenAIClient.
-        /// </summary>
         [RecordedTest]
-        public void InstanceTest()
+        [TestCase(OpenAIClientServiceTarget.Azure)]
```

Comment:
This is awesome, that tests can run for both cases!

Reply:
Yeah!! Many thanks to @m-nash for walking through how to do this reasonably with me; there's an even better version, but the `[TestCase()]` attributes are already an enormous improvement.